docs/content/dagster-cloud/account/managing-users/managing-user-roles-permissions.mdx

```diff
@@ -0,0 +1,748 @@
+      </td>
+    </tr>
+  </tbody>
+</table>
+
+### Applying role overrides
+
+<Note>This section is applicable to Dagster Cloud Enterprise.</Note>
+
+As previously mentioned, you can define default user roles for users in your organization. Dagster Cloud also supports applying overrides to specific code locations in a deployment or individual team members (if using [teams](/dagster-cloud/account/managing-users/managing-user-teams)).
+
+Overrides may be used to apply a **more permissive** role. If, for example, the default role is **Admin** or **Organization Admin**, overrides will be disabled as these are the most permissive roles.
+
+#### Code locations
+
+To override a code location role for an individual user:
+
+1. Locate the user in the list of users.
+2. Click **Edit**.
+3. Click the toggle to the left of the deployment to open a list of code locations.
+4. Next to a code location, click **Edit user role**.
+5. Select the user role for the code location: <Image
+   alt="Overriding the Viewer user role for a code location"
+   src="/images/dagster-cloud/user-token-management/code-location-override.png"
+   width={700}
+   height={499}
+   />
+6. Click **Save**.
+
+#### Team members
+
+Users in your organization can belong to one or more [teams](/dagster-cloud/account/managing-users/managing-user-teams). When determining a user's level of access, Dagster Cloud will use the **most permissive** role assigned to the user between all of their team memberships and any individual role grants.
+
+For example, let's look at a user with the following roles for our `dev` deployment:
+
+- **Team 1**: Launcher
+- **Team 2**: Viewer
+- **Individual**: Viewer
+
+In this example, the user would have **Launcher** access to the `prod` deployment. This is because the Launcher role is more permissive than Viewer.
+
+The above also applies to code locations and Branch Deployment roles.
+
+#### Viewing overrides
+
+To view deployment-level overrides for a specific user, locate the user on the **Users** page and hover over a deployment:
+
+<Image
+alt="Deployment overrides popup in Dagster Cloud"
+src="/images/dagster-cloud/user-token-management/user-overrides-popup.png"
+width={1010}
+height={265}
+/>
+
+If there are code location-level overrides, a small **N override(s)** link will display beneath the user's deployment role. Hover over it to display the list of overrides:
+
+<Image
+alt="Code location overrides popup in Dagster Cloud"
+src="/images/dagster-cloud/user-token-management/code-location-override-popup.png"
+width={1976}
+height={554}
+/>
+
+#### Removing overrides
+
+1. Locate the user in the list of users.
+2. Click **Edit**.
+3. To remove an override:
+   - **For a deployment**, click **Edit user role** next to the deployment.
+   - **For a code location**, click the toggle next to the deployment to display a list of code locations. Click **Edit user role** next to the code location.
+4. Click the **Remove override** button.
+5. Click **Save**.
```

Comment:
this only works for code location overrides right? we also refer to giving somebody an individual-level permission as an 'override' above

Reply:
This whole section? I'm able to do this for deployments and code locations in dogfood
![Screen Shot 2023-05-16 at 11 36 33 AM](https://github.com/dagster-io/dagster/assets/16615212/43c02e41-223b-49a5-96b9-c1163b2e1e69)